packages/utilities/src/object.ts

```diff
@@ -77,6 +79,9 @@ export function filteredAssign(isAllowed: (propName: string) => boolean, target:
   return target;
 }
 
+// Configure ids to reset on stylesheet resets.
+Stylesheet.getInstance().onReset(resetIds);
```

Comment:
I think perhaps a better pattern is through an eventemitter or something to decouple the utils from styling.

Reply:
Utils is already dependent on merge-styles which contains Stylesheet.